static/js/hotkey.js

```diff
@@ -532,6 +544,23 @@ exports.process_hotkey = function (e, hotkey) {
             return true;
         }
 
+        if (event_name === "narrow_compose" && compose_state.composing()) {
+            if (compose_state.get_message_type() === 'stream' && compose_state.stream_name() !== '') {
+                var stream = compose_state.stream_name();
+                var topic = compose_state.subject();
+                narrow.by_stream_topic_name(stream, topic);
+                compose_actions.start("stream");
```

Comment:
After narrowing to the required page the compose box closes. Compose box remains open only for pm-with narrow.

Reply:
We should figure out a way not to close the compose box in the `narrow` code, then, because closing and then re-opening may cause flickering.  There may be ways to do this with the existing functions, or, if not, we should refactor accordingly in a prep commit.